com.qq.QQmusic.yml

```
@@ -0,0 +1,72 @@
+app-id: com.qq.QQmusic
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/qqmusic.sh
+separate-locales: false
+
+finish-args:
+    - '--share=ipc'
+    - '--share=network'
+    - '--socket=x11'
+    - '--socket=pulseaudio'
+    - '--device=dri'
+    - '--device=shm'
+    - '--filesystem=/tmp'   # for singleton
+    - '--talk-name=org.freedesktop.Notifications'
+    - '--own-name=org.kde.*'
+    - '--own-name=org.mpris.MediaPlayer2.chromium.*'
+
+build-options:
+    strip: false
+    no-debuginfo: true
+
+modules:
+    -
+        name: qqmusic
+        only-arches:
+            - x86_64
+        sources:
+            -
+                type: script
+                commands:
+                    - /app/extra/QQmusic/qqmusic --no-sandbox
```

```suggestion
                    - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
                    - exec zypak-wrapper /app/extra/opt/QQmusic/qqmusic "$@"
```

This may need double-checking due to quotes.